lib/src/old_extended_nested_scroll_view.dart

```diff
@@ -723,7 +730,8 @@ class _NestedScrollCoordinator
       }
     }
 
-    if (innerPosition == null) {
+    // Kenshin: if innser scrollview is scrolled beyond top, change _outerPosition only
+    if (innerPosition == null || (_stretchHeaderSlivers && innerPosition.pixels == 0.0)) {
```

Comment:
this should be innerPosition.pixels <= 0.0, if use equal, sometimes, inner scrollview is not scroll to top

Reply:
thanks!